hack/cluster-clean.sh

```diff
@@ -98,7 +98,7 @@ function delete_namespaces() {
 function main() {
     echo "Cleaning up ..."
 
-    local kubevirt_managed_namespaces=(${namespace} ${cdi_namespace})
+    local kubevirt_managed_namespaces=(${namespace})
```

Comment:
nit, not mandatory
since $kubevirt_managed_namespaces now equal to $namespace
this var is unneeded

Reply:
@oshoval On the other hand if you remove `kubevirt_managed_namespaces` and use `namespace` instead you will have a variable which is not self explanatory as `kubevirt_managed_namespaces`. Maybe we should refactor the variable name `namespace` but then it's out of the PR scope, WDYT?

Reply:
I don't think it deserves aliasing for that reason,
personally i am also fine with keeping namespace as its the namespace which act as the cluster-clean scope